src/shared/components/mutationMapper/MutationMapperStore.ts

```diff
@@ -130,6 +130,37 @@ export default class MutationMapperStore
         }
     }, []);
 
+    readonly ptmData = remoteData({
+        await: () => [
+            this.mutationData
+        ],
+        invoke: async () => {
+            if (this.activeTranscript) {
+                return fetchPtmData(this.activeTranscript);
+            }
+            else {
+                return [];
+            }
+        },
+        onError: (err: Error) => {
+            // fail silently
+        }
+    }, []);
+
+    readonly ptmDataByProteinPosStart = remoteData<{[pos: number]: PostTranslationalModification[]}>({
+        await: () => [
+            this.ptmData
+        ],
+        invoke: async() => this.ptmData.result ? groupPtmDataByPosition(this.ptmData.result) : {}
```

Comment:
is it really possible that there ptmData.resul would be falsey even when the promise is complete?

Reply:
Hmm, probably not, unless service returns a null response without an error.